### ISSUE TITLE
Rename -v flag in mkdir command

### ISSUE THREAD

**Issue by cepessh** [embox/embox#3230]
In the current version of ```mkdir``` passing ```-v``` flag to the command means creating a virtual directory. As ```-v``` flag is widely used as a shortcut for ```--verbose```, I suggest to rename it to ```-s/--symbolic``` just like in ```ln``` command.

**Comment by Shankari02** [comment]
Even I feel this could be a good change as it will give better clarity and users can differentiate between the both easily.

**Comment by anton-bondarev** [comment]
> Even I feel this could be a good change as it will give better clarity and users can differentiate between the both easily.

Could you try to do this?

**Comment by Shankari02** [comment]
Yes, I have implemented it locally already. Will test it once and generate a pull request in a day for sure!
